docs/develop-guide.rst

```diff
@@ -336,38 +336,109 @@ Start the CLI to connect to the server and run SQL queries:
 7 How to run simple examples with Prestodb in distributed environment
 ---------------------------------------------------------------------
 
-(1) Create a folder to install cider files, for example ``cider``
-
-(2) | Copy the lib folder under the cider docker build environment to
-      every node, for example, copy
-      ``/usr/local/lib/`` folder to
-      ``/path/to/cider`` on every Prestodb node
+7.1 build presto in container
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 ::
 
-   cp -a /usr/local/lib/ /path/to/cider
+   git clone --recursive https://github.com/Intel-bigdata/presto.git
+   pushd presto
+   git checkout -b BDTK origin/BDTK
+   pushd presto-native-execution
+   git clone --recursive https://github.com/intel/BDTK.git
```

Comment:
`git clone --recursive https://github.com/intel/BDTK.git velox-plugin` then you don't need change include path in presto

Reply:
Now our project name is BDTK, why use the folder name velox-plugin?